src/components/Inbox/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import Message from "../Message";
+import {
+  Body, Container, Footer, Title,
+  Header, Close, SubTitle,
+} from "./styled";
+import "../../style.css";
+//@ts-ignore
+import close from "./close.svg";
```

Comment:
these are going to have issues during the build step, we will need a babel svg plugin

Reply:
how did we solve this one?